app/domain/models/order.py

```
@@ -32,22 +35,157 @@ class Order:
+    @property
+    def average_price(self) -> Decimal | None:
+        """평균 체결가 계산"""
+        if self.ord_type == OrderType.MARKET:
+            # 시장가 주문의 경우 실제 체결 금액에서 계산
+            if self.executed_volume > 0:
+                if self.side == OrderSide.BID and self.price:
+                    # 매수: price는 총 지불 금액
+                    return self.price / self.executed_volume
+                elif self.side == OrderSide.ASK and self.price:
+                    # 매도: 별도 계산 필요
+                    return self.price
```

Pylint (3.3.7)

[refactor] Unnecessary "elif" after "return", remove the leading "el" from "elif"

(R1705)